src/bootstrap-customisations/forms/_form-control.scss

```diff
@@ -27,6 +27,21 @@ textarea.form-control {
 }
 .invalid-feedback {
     font-family: $fhi-font-family-sans-serif-medium;
+    margin-bottom: 0;
```

Comment:
As mentioned, a little bit skeptic when it comes to these width-classes...

Reply:
> As mentioned, a little bit skeptic when it comes to these width-classes...

Me too, so, do we ditch them for now?

Reply:
I think; yes...